best-practices/README.md

```diff
@@ -264,6 +264,21 @@ Objective-C
 * Prefer categories on `Foundation` classes to helper methods.
 * Prefer string constants to literals when providing keys or key paths to methods.
 
+Android
+-------
+
+* Each callback on an `Activity` should invoke methods on an instance of a
+  presenter object.
+* Unit test each presenter object.
+* Integration tests are optional and not encouraged.
+* Always prefix `R.id` with the `R`, even if Android Studio tries to remove it.
+* Divide packages by feature, not design pattern. The same package will likely
+  have one activity and one presenter.
+* Do not use `android:onClick`.
```

Comment:
maybe specify `in XML`

Reply:
I also wonder for advice like this if it's worth including a "why"? not sure if that's out of the scope of `best practices` but my first instinct when i read something like "do not do this" is to ask "why?"